_posts/2023-05-07-observable-state-in-angular.md

```diff
@@ -540,6 +534,13 @@ npx nx run shop:serve
 public onlySelectWhen(keys: (keyof T)[]): Observable<T>;
 ```
 
+That's it. You can find a complete example [here](https://github.com/simplifiedcourses/observable-state/blob/main/src/demo-app/components/smart/product-overview/product-overview.smart-component.ts){:target="_blank"}. If you clone the repo, you can run it with:
```

Comment:
Move this to the bottom because I found it weird that this was before the API section 😅
You could also include this within "Wrapping up"